doc/content/modules/user-manual/pages/release-notes/2025.4.0.adoc

```diff
@@ -212,6 +212,8 @@ action a0 {
 }
 ```
 - Implement `ConnectionDefinition` derived references and operations in SysON SysMLv2 metamodel.
+- `TransitionUsages` can now de displayed and created in the _General View_ and _Action Flow_ diagrams.
+Be aware, that the guard expressions are displayed in the label but *can't* be edited using the direct edit tool.
```

Comment:
Be aware, that => Be aware that

*can't* be edited using  => *can't* be edited yet using

Reply:
Done